Libbna/CUSTOM-CMS
# #198: Fixed the issue with redirection, now the logo appear without refresing the page [WIP]

Steps to test:

1. Login with admin access
2. Goto config page
3. Add a new logo
4. Check whether the logo is changed immediately after clicking on the submit button

Note:- The message is still not getting displayed*
